### PR TITLE
Spanish Changes and Some Fixes

### PR DESCRIPTION
**Pull request by brosscode**
Known Bugs that are WIP:

Flag does not align on Navbar and other css issues
Cannot swap back to English
Does not Automatically detect browser language
Icon missing above the footer when language is changed

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -65,7 +65,9 @@ body {
   font-size: 3em;
 }
 
-#testimonial-text {
+#testimonial-text-nic,
```
Comment (ForzenAcademy) [inline css/styles.css]: Why not just make this a class instead of separating it?

File: css/styles.css [hunk 2]
```diff
@@ -122,7 +124,10 @@ body {
   font-size: 0.3em;
 }
 
-#get-your-quote {
+#auto-get-your-quote,
```
Comment (ForzenAcademy) [inline css/styles.css]: Why not class?

File: js/scripts.js [hunk 3]
```diff
@@ -458,4 +458,314 @@ function tabClick(tab) {
...
+        document.getElementById("day-life-err"),
+        document.getElementById("year-life-err"),
+        document.getElementById("address-one-life-err"),
+        document.getElementById("city-life-err"),
+        document.getElementById("state-life-err"),
+        document.getElementById("zip-life-err"),
+        document.getElementById("email-address-life-err"),
+        document.getElementById("phone-number-life-err"),
+    ]
+    var spanishErrorText = "Campo requerido" // Field Required
```
Comment (ForzenAcademy) [inline js/scripts.js]: Never var in js, always `let`

**Review by ForzenAcademy — COMMENTED** [review]
Please make the flag two SEPARATE images, and not huge ones that are 20KB please

File: css/styles.css (hunk 1 above)
Comment (brosscode) [reply css/styles.css]: Changed to a class

File: css/styles.css (hunk 2 above)
Comment (brosscode) [reply css/styles.css]: also changed it to a class

File: js/scripts.js (hunk 3 above)
Comment (brosscode) [reply js/scripts.js]: changed to let

**Review by ForzenAcademy — APPROVED** [review]

**Pull request merged**